contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -280,112 +180,107 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, Reentrancy
         }
     }
 
-    /// @notice Allows to claim revenue tokens on behalf of the ip royalty vault by snapshot batch
-    /// @param snapshotIds The list of snapshot ids
+    /// @notice Get total amount of revenue token claimable by a token holder
```

Comment:
Get total amount of revenue token claimable by a token holder

could be for clarity

Get total amount of revenue token claimable by a royalty token holder

Reply:
changed